src/main/java/htsjdk/tribble/readers/TabixReader.java

```diff
@@ -56,6 +56,9 @@
     private int mBc;
     private int mEc;
     private int mMeta;
+    /** longest line seen so far, used to allocate the capacity of the lines */
+    private int longestLineLength = 100;
```

Comment:
Could you separate out this change to the longest line?  It seems orthogonal to the rest of the changes.  I see how it could potentially save us some allocations, but it adds complexity with somewhat questionable return.  If you want to change it to just set the default buffer size to something greater than 16, we could merge that as part of this PR.  Otherwise lets do that in a separate PR.